src/libraries/System.Data.Common/src/System/Data/DataTable.cs

```diff
@@ -6704,7 +6708,12 @@ public static XmlSchemaComplexType GetDataTableSchema(XmlSchemaSet? schemaSet)
             return XmlSchema.Read(new XmlTextReader(stream), null);
         }
 
-#pragma warning disable 8632
+        [RequiresUnreferencedCode("DataTable.GetSchema uses TypeDescriptor and XmlSerialization underneath which are not trimming safe. Members from serialized types may be trimmed if not referenced directly.")]
+        private XmlSchema? GetXmlSchema()
```

Comment:
Who calls this?

Reply:
good catch, `IXmlSerializable.GetSchema()` was supposed to